src/midi/midi_alsa.cpp

```diff
@@ -45,7 +45,7 @@ static void for_each_alsa_seq_port(port_action_t action)
 	// be called before that sequencer is created.
 	snd_seq_t *seq = nullptr;
 	if (snd_seq_open(&seq, "default", SND_SEQ_OPEN_OUTPUT, 0) != 0) {
-		LOG_MSG("ALSA: Error: Can't open MIDI sequencer");
+		LOG_WARNING("MIDI:ALSA: Can't open MIDI sequencer");
```

Comment:
`MIDI:ALSA:` -> nice clarification! Especially useful because there is a big difference in how ALSA handles MIDI stream versus PCM (and diagnosing them is quite a different task.. with different kernel modules underpinning each).